debian/changelog

```diff
@@ -1,3 +1,14 @@
+aad-auth (0.3) kinetic; urgency=medium
+
+  * Add aad-cli binary package (LP: #1989413)
+    - this is a CLI that allows managing various aspects of the AAD
+      integration, such as:
+      - editing the configuration file in a safe way
+      - querying cached users/groups attributes
+      - updating cached user attributes (shell/homedir)
```

Comment:
You should mention update debian/copyright updates, also there are some other changes in this release, please list them :)